### ISSUE TITLE
Add h1 to header's page title on a site's homepage

### ISSUE THREAD

**Issue by trubach** [ATLAS-Illinois/Illinois-Framework-Theme#1189]
On a site's homepage, the page's title in the `<ilw-header>` component should be wrapped in an `<h1>` tag. This will address an accessibility issue where the h1 tag doesn't match the title in the `<title>` tag for the page. This behavior should only happen on the site's homepage. Other pages on the site will use Drupal's title field as the `<h1>`, which is shown at the top of the main content.

**Describe the solution you'd like**
The code for the homepage should look like this:
<img width="1340" height="1308" alt="Image" src="https://github.com/user-attachments/assets/78ef8c5b-c347-49a1-9166-96ba4fc297b7" />

**Additional context**
This partially addresses an issue reported by Tim Offenstein where the content in the page's `<title>` tag doesn't match what's in the `<h1>` tag.
